boards/aithinker/ai_wb2_12f/ai_wb2_12f_defconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024-2025 MASSDRIVER EI (massdriver.space)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+
+CONFIG_UART_CONSOLE=y
+CONFIG_UART_BFLB=y
```

Comment:
```suggestion
```

Reply:
Cant, 'default y' is missing from the KConfig.

Reply:
I added a commit to fix it actually.

Reply:
Cool, thanks! You should also drop it from bl604dvk defconfig while you're here.

Reply:
Done